src/commands/projects.ts

```diff
@@ -131,22 +131,50 @@ export const handler = (args: yargs.Argv) => {
 };
 
 const list = async (props: CommonProps) => {
-  const result: ProjectListItem[] = [];
-  let cursor: string | undefined;
-  let end = false;
-  while (!end) {
-    const { data } = await props.apiClient.listProjects({
-      limit: PROJECTS_LIST_LIMIT,
-      cursor,
-    });
-    result.push(...data.projects);
-    cursor = data.pagination?.cursor;
-    log.debug('Got %d projects, with cursor: %s', data.projects.length, cursor);
-    if (data.projects.length < PROJECTS_LIST_LIMIT) {
-      end = true;
+  const getList = async (
+    fn:
+      | typeof props.apiClient.listProjects
+      | typeof props.apiClient.listSharedProjects,
+  ) => {
+    const result: ProjectListItem[] = [];
+    let cursor: string | undefined;
+    let end = false;
+    while (!end) {
+      const { data } = await fn({
+        limit: PROJECTS_LIST_LIMIT,
+        cursor,
+      });
+      result.push(...data.projects);
+      cursor = data.pagination?.cursor;
+      log.debug(
+        'Got %d projects, with cursor: %s',
+        data.projects.length,
+        cursor,
+      );
+      if (data.projects.length < PROJECTS_LIST_LIMIT) {
+        end = true;
+      }
     }
-  }
-  writer(props).end(result, { fields: PROJECT_FIELDS });
+
+    return result;
+  };
+
+  const lists = await Promise.all([
```

Comment:
nit: you can use destructuring to make the result more verbose
```suggestion
  const [ownedProjects, sharedProjects] = await Promise.all([
```